packages/fast-components-class-name-contracts-base/src/checkbox/index.ts

```diff
@@ -2,9 +2,28 @@
  * The class name contract for the checkbox component
  */
 export interface ICheckboxClassNameContract {
+    /**
+     * The root checkbox component
+     */
     checkbox: string;
-    checkbox_disabled: string;
+
+    /**
+     * The disabled state modifier
+     */
+    checkbox__disabled: string;
+
+    /**
+     * The checkbox input element
+     */
     checkbox_input: string;
+
+    /**
+     * The label associated with the checkbox
+     */
     checkbox_label: string;
-    checkbox_span: string;
+
+    /**
+     * The checkbox state indicator (e.g. checked, inditerminant, or not checked)
```

Comment:
I'd omit the "eg" here - to me that is more confusing because it speaks to the *state* while the sentence speaks to the *element*

Reply:
+1 and spelling of `indeterminate`